### PR TITLE
updated landing page.

### PR DESCRIPTION
**Pull request by EfrataAron**
i have removed the unnecessary files. landing page has been worked on.

### CODE REVIEW

File: playground-1.mongodb.js
Comment (notsoocool) [inline playground-1.mongodb.js:1]: what is the point of this file?

Comment (EfrataAron) [reply playground-1.mongodb.js:1]: actually its not supposed to be there

Comment (EfrataAron) [reply playground-1.mongodb.js:1]: you can remove it.

Comment (EfrataAron) [reply playground-1.mongodb.js:1]: @notsoocool i have fixed that issue. any other ?

**Pull request closed without merging**